src/organizations/components/OrgProfileTab/DeletePanel.tsx

```diff
@@ -94,8 +96,10 @@ const DeleteOrgButton: FC = () => {
         dispatch(notify(deleteOrgWarning(buttonElement)))
       } else {
         dispatch(
-          showOverlay('suspend-org-in-paid-account', null, () =>
-            dispatch(dismissOverlay())
+          showOverlay(
+            'suspend-org-in-paid-account',
+            {users: users.length},
+            () => dispatch(dismissOverlay())
```

Comment:
Let's call this something like `userCount`, since it's just a number instead of a user object.